glances/plugins/containers/model.py

```diff
@@ -76,13 +76,13 @@ def __init__(self, args=None, config=None):
         self.display_curse = True
 
         # Init the Docker API
-        self.docker_extension = DockerContainersExtension() if not import_docker_error_tag else None
+        self.docker_extension = DockerContainersExtension(server_urls=self._docker_sock()) if not import_docker_error_tag else None
 
         # Init the Podman API
         if import_podman_error_tag:
             self.podman_client = None
         else:
-            self.podman_client = PodmanContainersExtension(podman_sock=self._podman_sock())
+            self.podman_client = PodmanContainersExtension(podman_sock=self._podman_sock()) # TODO: podman also
```

Comment:
IS it also possible to have multiple socket for Podman ?

Reply:
@nicolargo 
Yep.
In fact its more common to have custom socket's in the podman ecosystem.